cloudify_cli/VERSION

```diff
@@ -1,4 +0,0 @@
-{
-  "edition": "premium",
-  "version": "5.1.0-.dev1"
-}
```

Comment:
This will affect the release process for both community and premium,

Reply:
@mabuaisha where?

Reply:
In the build-system but I think we should be fine since there is no something specific to cli defined inside the config.yaml https://github.com/cloudify-cosmo/cloudify-build-system/blob/master/config/core-config.yaml#L82

And I think for community we are bump the community version inside `setup.py` which I think will be fine https://github.com/cloudify-cosmo/cloudify-cli/blob/20.03.03-build/setup.py#L21

Reply:
yes, good, so this doesn't affect anything after all

and bumping this file can be removed as well of course

Reply:
or bumping doesn't need to be removed (yet) because i think we still have a VERSION file somewhere? (rest?)

Reply:
well, it search for this regex https://github.com/cloudify-cosmo/cloudify-build-system/blob/master/config/core-config.yaml#L26 which means if this file does not exist on the cli then it will try to match on agent and I think there is another one on cloudify-manager-install 

I think this should be fine because at the end of the day this file used in order to grap the version of the current cli and since you depend on python to get that version then we are fine I think